test/contributors/annotations/data_class/data_class_assist_contributor_test.dart

```diff
@@ -2,8 +2,8 @@ import 'dart:io' as io;
 
```

Comment:
Data class annotation tests failed for me.
They expect in-place generation, but got a generated file instead.